### PR TITLE
Hp/create sign in

### PR DESCRIPTION
**Pull request by hernan-dp**
Sign in and sign up layouts finished

### CODE REVIEW

File: src/Components/SignIn.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import React, {Component} from "react";
```
Comment (abelosorio) [inline src/Components/SignIn.js]: Make sure to run the linter before creating the PR.

File: src/Components/SignIn.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+  handleChange = (e) =>{
+    this.setState({
+      [e.target.id]: e.target.value
+    });
+  }
+
+  handleSubmit = (e) => {
+    e.preventDefault();
+    console.log(this.state.username, this.state.password);
```
Comment (abelosorio) [inline src/Components/SignIn.js]: Make sure to remove debug messages when you're ready with testing.

File: src/Components/SignIn.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        <div className='middle'>
+          <div className='inner'>
+            <div className='wrapper fadeInDown'>
+              <div id='formContent'>
+                <h1>Sign In</h1>
+                <form onSubmit={this.handleSubmit}>
+                  <input type='text' id='username' className='fadeIn first' placeholder='Username' onChange={this.handleChange} />
+                  <input type='password' id='password' className='fadeIn second' placeholder='Password' onChange={this.handleChange} />
+                  <button type='submit' className='fadeIn third' onClick={this.handleSubmit}>Login</button>
+                </form>
```
Comment (nahudalla) [inline src/Components/SignIn.js]: I recommend that you use a library such as Formik for form state management.

File: src/Components/SignUp.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+            <div className='wrapper fadeInDown'>
+              <div id='formContent'>
+                <h1>Member Registration</h1>
+                <form onSubmit={this.handleSubmit}>
+                  <input type='text' id='firstname' className='fadeIn first' placeholder='FirstName' onChange={this.handleChange} />
+                  <input type='text' id='lastname' className='fadeIn second' placeholder='LastName' onChange={this.handleChange} />
+                  <input type='text' id='username' className='fadeIn third' placeholder='Username' onChange={this.handleChange} />
+                  <input type='password' id='password' className='fadeIn fourth' placeholder='Password' onChange={this.handleChange} />
+                  <button type='submit' className='fadeIn third' onClick={this.handleSubmit}>Sign up</button>
+                </form>
```
Comment (nahudalla) [inline src/Components/SignUp.js]: Same here about the form state management.

**Comment by mpautasso** [comment]
@hernan-dp Could you apply @nahudalla suggestions, please?

**Pull request merged**